tools/xcodeproj_shims/installers/indexstores.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 
 echo "Start remapping index files at `date`"
 
-FOUND_INDEXSTORES=`pcregrep -o1 'command_line: "(.*\.indexstore)' $BAZEL_BUILD_EVENT_TEXT_FILENAME || true`
+FOUND_INDEXSTORES=`pcregrep -o1 'command_line: "(.*\.indexstore)' $BAZEL_BUILD_EVENT_TEXT_FILENAME | uniq`
```

Comment:
will this now fail if no commandline matches the pattern?

Reply:
i tested this locally, and it seems to just result in an empty array and an exit status code of 0:

```
~/Development/register master * fi6=`pcregrep -o1 'command_line: "(.*\.indexstore)' /tmp/foo | uniq`; echo $?
0
~/Development/register master * cat /tmp/foo
hi
```